engine/overworld/movement.asm

```diff
@@ -64,18 +64,18 @@ UpdatePlayerSprite:
 	ld a, [H_CURRENTSPRITEOFFSET]
 	add $7
 	ld l, a
-	ld a, [hl]
+	ld a, [hl] ; $c1x7 (IntraAnimFrameCounter)
 	inc a
-	ld [hl], a
+	ld [hl], a ; $c1x7 (IntraAnimFrameCounter)
```

Comment:
Rather than all these comments, the `add 7` should probably just be changed to
`add PlayerIntraAnimFrameCounter - PlayerSpriteStateData1`,
or add a constant and do `add SPRITESTATE_INTRA_ANIM_FRAME`.